server/game/cards/03_TWI/units/LethalCrackdown.ts

```diff
@@ -12,22 +12,21 @@ export default class LethalCrackdown extends EventCard {
 
     public override setupCardAbilities() {
         this.setEventAbility({
-            title: 'Defeat a non-leader unit. Deal damage to your base equal to that unit\'s power.',
+            title: 'Defeat a non-leader unit.',
             targetResolver: {
                 cardTypeFilter: WildcardCardType.NonLeaderUnit,
                 immediateEffect: AbilityHelper.immediateEffects.sequential([
                     AbilityHelper.immediateEffects.handler((context) => ({
-                        handler: () => context.targets.powerParentCard = context.target.getPower(),
+                        handler: () => context.targets.cardPower = context.target.getPower(),
```

Comment:
Sorry for the last-minute change, but I finally merged in the fix that means we don't have to use `handler` anymore in this situation. Can you merge main and look at the updated Rose Tico implementation and reference that? The basic idea is that you can just do the defeat here and then read the power in `ifYouDo` as `ifYouDoContext.events[0].lastKnownInformation.power`